test/CMakeLists.txt

```diff
@@ -15,6 +15,18 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+# If TBB headers are available, libstdc++ enables TBB backend; some TBB symbols become undefined unless it is linked.
+# The undefined symbol error arises only with GCC compiler.
+if (CMAKE_CXX_COMPILER_ID STREQUAL GNU AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 9)
```

Comment:
Even though libstdc++ uses TBB starting with v9, we manually disable it for GCC versions 9 and 10 in `dpl/internal/common_config.h` if oneTBB is found. See: https://github.com/oneapi-src/oneDPL/blob/a70f043558df5c054e1ffe1664d839097e253db1/include/oneapi/dpl/internal/common_config.h#L37

I ran the following simple test with GCC versions 9, 10, 11 and 12 and only saw linker errors with versions 11 and above:
```bash
cmake -DCMAKE_CXX_COMPILER=g++ -DONEDPL_BACKEND=serial .. && make sycl_iterator_for.pass
```
Should we change this 9 to 11?

Reply:
To reproduce the issue with libstdc++ 9/10, you need TBB 2020 or older.

The referenced workaround turns off TBB usage in libstdc++ 9/10 if oneTBB is found. This is because libstdc++ 9/10 use old TBB API. oneTBB is not backward compatible with TBB: code compiled with TBB headers and linked against oneTBB will not generally work.

This issue is possible with versions starting with "9". However, as far as I know, only oneTBB is supported. I am slightly inclined toward leaving "9", but I can be convinced change it to "11".

Reply:
The version has been changed to 11 after https://github.com/oneapi-src/oneDPL/pull/553#discussion_r1373141337 discussion.